tutorials/checker/build.gradle.kts

```diff
@@ -78,18 +78,48 @@ fun checkDirs(dirs: List<String>, template: String) {
 // NOTICE: currently we use a bit hacky approach, when "```kotlin" marks code that shall be checked, while "``` kotlin"
 // with whitespace marks code that shall not be checked.
 tasks.register("check") {
+  val checks = CheckSpec.createCheckSpecs(
+    checkTargets = (project.property("CHECK_TARGET")?.toString() ?: "all").toLowerCase()
```

Comment:
toLowerCase is obsolete in Kotlin 1.5

Reply:
Apparently , even latest gradle 7.1 is using kotlin stdlib 1.4.31 where `lowercase()` function is experimental, so it requires adding extra @OptIn annotation. I think we'll change it to `lowercase()` as soon as gradle starts using kotlin stdlib 1.5 or newer. 

<img width="200" alt="Screenshot 2021-06-29 at 16 38 40" src="https://user-images.githubusercontent.com/7372778/123817833-c72c7280-d8f8-11eb-8eba-cd0040e16fe4.png">